presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -428,6 +428,16 @@ public void testCast()
         assertQuery("SELECT CAST(true as VARCHAR), CAST(false as VARCHAR)");
         assertQuery("SELECT CAST(0.0 as VARCHAR)");
 
+        // Cast to varchar(n).
+        assertQuery("SELECT CAST('string_longer_than_1_character' AS VARCHAR(1)");
+        assertQuery("SELECT CAST(comments as VARCHAR(1)) FROM orders)");
+        assertQuery("SELECT CAST('string_shorter_than_1000_character' AS VARCHAR(1000)");
+        assertQuery("SELECT CAST(comments as VARCHAR(1000)) FROM orders WHERE LENGTH(comment) < 1000");
```

Comment:
Does this data have comments > varchar(1000) ? So there might not be any truncation.

Reply:
Yes, this test case is to test no truncation for varchar values shorter than 1000 if the max length of return type is set to 1000.